src/Core_Command.php

```diff
@@ -764,8 +788,10 @@ private static function modify_wp_config( $content ) {
 
 	private static function get_clean_basedomain() {
 		$domain = preg_replace( '|https?://|', '', get_option( 'siteurl' ) );
-		if ( $slash = strpos( $domain, '/' ) )
+		$slash  = strpos( $domain, '/' );
+		if ( $slash ) {
```

Comment:
This is a bug. Please make this a strict comparison. `strpos()` can return (int) 0 or (bool) `false` and they really have different meanings.
Ref: https://www.php.net/manual/en/function.strpos.php